docs/design.md

```diff
@@ -1,30 +1,28 @@
-# PromLTS
+# Design
 
-PromLTS is a set of components that can be composed into a highly available Prometheus setup with long term storage capabilities. Its main goals are operation simplicity and retaining of Prometheus's reliability properties.
+Thanos is a set of components that can be composed into a highly available Prometheus setup with long term storage capabilities. Its main goals are operation simplicity and retaining of Prometheus's reliability properties.
 
-Unlike existing solutions it does not attempt to integrate with the official yet experiemental remote APIs. This has a variety of reasons, which go beyond the scope of this document.
-Instead its main integration point is the Prometheus 2.0 storage format ([spec][tsdb-format], [slides][tsdb-talk]) itself through the storage engine's [library][tsdb-lib].
+The Prometheus metric data model and the 2.0 storage format ([spec][tsdb-format], [slides][tsdb-talk]) are the foundational layers of all components in the system.
 
 ## Architecture
 
-### Shipping Data
+Thanos is a clustered system of components with distinct and decoupled purposes. Clustered components can be categorized as follows:
 
-```
-┌────────────┬─────────┐                   ┌────────────┬─────────┐
-│ Prometheus │ Sidecar │        ...        │ Prometheus │ Sidecar │
-└────────────┴────┬────┘                   └────────────┴────┬────┘
-                  │                                          │
-                Blocks                                     Blocks
-                  │                                          │
-                  v                                          v
-              ┌──────────────────────────────────────────────────┐
-              │               Google Cloud Storage               │
-              └──────────────────────────────────────────────────┘
-```
+* Data sources
+* Stores
+* Queriers
```

Comment:
What about other things? Compactor, ruler?

Reply:
Ruler is a data source. Compactor is not a clustered component. 
I tried to keep the design doc a bit more abstract and don't pin it down to specific implementation we provide too much as that's already covered sufficiently in component docs.

Reply:
ah, right